hydra/conf/__init__.py

```diff
@@ -122,6 +122,8 @@ class HydraConf:
     sweeper: Any = MISSING
     # Launcher configuration
     launcher: Any = MISSING
+    # Callbacks configuration
+    callbacks: Any = MISSING
```

Comment:
It would be better to make this an empty dictionary which is contributed to instead of MISSING.

Something like:
```python
callbacks: Dict[str, Any] = field(default_factory=dict)
```